compute_sdk/globus_compute_sdk/sdk/batch.py

```diff
@@ -2,13 +2,29 @@
 
 import typing as t
 from collections import defaultdict
+from dataclasses import asdict, dataclass
 
 from globus_compute_sdk.sdk.utils.uuid_like import UUID_LIKE_T
 from globus_compute_sdk.serialize import ComputeSerializer
 
 _default_serde = ComputeSerializer()
 
 
+@dataclass
+class UserRuntime:
+    """Information about a user's runtime environment, which is sent along with task
+    submissions to MEP user config templates.
+
+    :param str globus_compute_sdk_version: Version of the Compute SDK
+    :param str globus_sdk_version: Version of the Globus SDK
+    :param str python_version: Python version running the Compute SDK
```

Comment:
How does this show in the built documentation?  Now that we have typing as a first-class citizen within the Python ecosystem (something we didn't have 15+ years ago), I've been intentionally leaving the type out of the documentation string so that there is a single point of authority.

Not a blocker, but a suggestion.

Reply:
Without the explicit types, the types don't show up in the documentation site. Not sure exactly why, but probably has to do with the fact that I'm documenting essentially the constructor of the `dataclass` by using a docstring at the class level.